src/components/Boards/components/BoardTile/BoardTile.styles.tsx

```diff
@@ -0,0 +1,25 @@
+import { styled } from '@material-ui/core/styles';
```

Comment:
We can pull this out into shared styled components in the future.

Reply:
What's the purpose of the muted link? Is this a stylistic choice? Let's make sure the app is still accessible

Reply:
This just removes text decoration, shouldn't affect accessibility.

Reply:
👍